provisioning_client/adapters/hsm_client_riot.c

```diff
@@ -780,7 +785,7 @@ char* hsm_client_riot_create_leaf_cert(HSM_CLIENT_HANDLE handle, const char* com
         LEAF_CERT_TBS_DATA.SubjectCommon = common_name;
 
         DERInitContext(&leaf_ctx, leaf_buffer, DER_MAX_TBS);
-        if (X509GetDeviceCertTBS(&leaf_ctx, &LEAF_CERT_TBS_DATA, &leaf_id_pub) != 0)
+        if (X509GetDeviceCertTBS(&leaf_ctx, &LEAF_CERT_TBS_DATA, &leaf_id_pub, NULL, 0) != 0)
```

Comment:
It appears we're creating a "leaf cert" but we're calling X509GetDeviceCertTBS.  This should instead be X509GetAliasCertTBS.

Reply:
Fixed, but @DMattoon PTAL specifically the function params